cypress/support/commands.js

```diff
@@ -0,0 +1,61 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+
+const severityIndicators = {
+  minor: '⚪️',
+  moderate: '🟡',
+  serious: '🟠',
+  critical: '🔴'
+}
+
+function callback(violations) {
+  violations.forEach(violation => {
+    const nodes = Cypress.$(violation.nodes.map(node => node.target).join(','))
+
+    Cypress.log({
+      name: `${severityIndicators[violation.impact]} A11y`,
+      consoleProps: () => violation,
+      $el: nodes,
+      message: `[$${violation.help}](${violation.helpUrl})`
+    })
+
+    violation.nodes.forEach(({ target }) => {
+      Cypress.log({
+        name: '🔧',
+        consoleProps: () => violation,
+        $el: Cypress.$(target.join(',')),
+        message: target
+      })
+    })
+  })
+}
+
+Cypress.Commands.add('checkPageA11y', (path) => {
+  cy.visit(path);
+  cy.injectAxe();
+  cy.checkA11y(null, null, callback);
+})
```

Comment:
Is this file needed since we'll be removing the .checkPageA11y  comment?

Reply:
Yes, this file is a default Cypress file and will break it if we remove it.